File: src/app/model/CommonUser.ts
export class CommonUser
{
    emailId?:any;
    firstName?:any;
    lastName?:any;
    password?:any;
    profilePicture?:any|null;
    constructor(emailId:any, firstName:any,password:any ,lastName:any,
        profilePicture:any){
            this.emailId = emailId;     
            this.firstName = firstName;
            this.password = password;
            this.profilePicture = profilePicture;
                  this.lastName = lastName
        }
}

File: User-Services/src/main/java/foodieapp/UserServices/domain/CommonUser.java
package foodieapp.UserServices.domain;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;
import org.springframework.data.annotation.Id;
import org.springframework.data.mongodb.core.mapping.Document;

@Document
@AllArgsConstructor
@NoArgsConstructor
@Data
public class CommonUser {
    @Id
    private String emailId;
    private byte[] profilePicture;
    private String firstName;
    private String lastName;
    private String password;
    private String role;
}
